overlord/patch/patch6_2_test.go

```diff
@@ -209,48 +209,6 @@ var statePatch6_2JSON = []byte(`
 	}
 }`)
 
-// State with 'snapd' snap with proper snap type, and an extra 'other'
-// snap with snapd snap-id (PMrrV4ml8uWuEUDBT8dSGnKUYbevVhc4) but
-// improper 'app' type.
-var statePatch6_2JSONWithSnapd = []byte(`
```

Comment:
Hmmmmm... Looks like an oversight (my mistake) when I introduced it. A bit of an edge case that is actually not handled correctly right now because a test using this definition is missing. Could you please keep this variable and I'll work on a separate PR with a fix?

Reply:
I proposed #10246 that uses this var and handles the related case.